update-docs-sources

```diff
@@ -10,6 +10,13 @@ for i in phpdoc-all; do
 	svn up -q
 done
 
+langs=`/usr/bin/php -r 'include "/local/src/phpweb/include/languages.inc"; echo implode( " ", array_keys( $ACTIVE_ONLINE_LANGUAGES ) );'`;
+langs="base $langs"
+for i in $langs; do
+  cd /local/src/phpdoc-git/$i
+  git pull
```

Comment:
Does `git pull` work correctly with force pushes? Possibly this should `git fetch origin && git checkout origin/master`?

Also, as written now this requires someone with access to the server to do the initial clone of the repos, right?

Reply:
That was always the case. but the repos are already existing and are already updated constantly on the machine (as this file is not ion sync between git and euk2. 

And yes: The current version of the file is doing a `git fetch origin && git reset --hard origin/master`. I will adapt the script accordingly